### PR TITLE
Recreating this branch without excess merging

### PR DESCRIPTION
**Pull request by krishnakpandian**
Problem
=======
C2D-26 [Jira](https://cruzhacks-dev.atlassian.net/browse/C2D-26)

Solution
========
What I/we did to solve this problem
* Implemented About Component
* Everything has been merged in together except the header

Change Summary:
---------------
* Updated About.tsx/scss, AboutBox.tsx/scss, other files are from merge conflicts 

Dev-Ops
=======
If you added an ENV variable, please check off that you've updated the following  
- [ ] Key & Sample value to `.env` & `sample.env` 
- [ ] GCP Secret Manager (Both development and production)
- [ ] Github Secrets (Added a development and production variable)
- [ ] Github Workflows `.github/workflows/dev.yml` & `.github/workflows/prod.yml`
- [ ] webpack-config.js

Images/Important Notes (optional):
-----------------------
* 

### Small
about-small

### Mobile
about-mobile

### Tablet
about-tablet

### Desktop
about-desktop

### CODE REVIEW

File: src/views/Home/components/About/AboutBox/AboutBox.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.about-box {
+    z-index: 1;
+    border-radius: 33px;
+    width: 80%;
+    min-height: 331px;
```
Comment (w-gao) [inline src/views/Home/components/About/AboutBox/AboutBox.scss]: We probably don't need this. It leaves a huge empty space on certain devices.

File: src/images/icons/about-background-line-desktop.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 1437 618" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (w-gao) [inline src/images/icons/about-background-line-desktop.svg]:
```suggestion
<svg preserveAspectRatio="none" viewBox="0 0 1437 618" fill="none" xmlns="http://www.w3.org/2000/svg">
```

If we want the background to stretch all the way when the height doesn't allow it (e.g.: on `big-desktop`s), consider adding this. I think it's fine to stretch these graphics.

File: src/views/Home/components/About/AboutBox/AboutBox.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        }
+    }
+    &__description {
+        font-family: Lato, "sans-serif";
+        line-height: 42px;
+        color: var(--color-text-primary);
+        height:100%;
+        border-bottom-left-radius: 33px;
+        border-bottom-right-radius: 33px;
+        padding: 1% 2% 2% 2%;
```
Comment (w-gao) [inline src/views/Home/components/About/AboutBox/AboutBox.scss]:
```suggestion
        padding: 1% 4% 2% 4%;
```
The padding looks small on desktop compared to the figma.

File: src/views/Home/components/About/AboutBox/AboutBox.scss [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        border-top-right-radius: 33px;
+        @include responsiveness.font($responsive: 4.1vw, $min: 35px, $max: 80px);
+        @include responsiveness.screen(smallest, phone) {
+            @include responsiveness.font($responsive: 7.1vw, $min: 22px, $max: 40px);
+            padding: 0 26px 0 30px;
+        }
+    }
+    &__description {
+        font-family: Lato, "sans-serif";
+        line-height: 42px;
```
Comment (w-gao) [inline src/views/Home/components/About/AboutBox/AboutBox.scss]: I don't think we need to set `line-height` actually, the default looks fine to me. The same goes for line 40.

File: src/views/Home/components/About/AboutBox/AboutBox.scss [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        color: var(--color-text-primary);
+        height:100%;
+        border-bottom-left-radius: 33px;
+        border-bottom-right-radius: 33px;
+        padding: 1% 2% 2% 2%;
+        @include responsiveness.font($responsive: 3.5vw, $min: 20px, $max: 35px);
+
+        @include responsiveness.screen(smallest, phone, tablet) {
+            padding: 0 26px 20px 30px;
```
Comment (w-gao) [inline src/views/Home/components/About/AboutBox/AboutBox.scss]:
```suggestion
            padding: 10px 26px 20px 30px;
```

Consider adding padding top to tablet as well (and remove lines 43 - 45).

**Review by w-gao — COMMENTED** [review]
This looks pretty good! Just some styling suggestions. Although, we can always tweak theses after our initial launch.

File: src/views/Home/components/About/index.scss [hunk 6]
```diff
@@ -0,0 +1,35 @@
+@use "./stylesheets/responsiveness";
+
+.about-component {
+    background: linear-gradient(#8296D9,#BFACF0, #8F9ADD, #D99CE3);
+    position: relative;
+    z-index:-1;
```
Comment (w-gao) [inline src/views/Home/components/About/index.scss]:
```suggestion
    z-index: 1;
```
Also, this shouldn't be -1. It's messing up the cursor for text selection on chrome.

Comment (w-gao) [reply src/views/Home/components/About/index.scss]: This can be 0 as well, since `&__background` is at -1.

**Review by Alperdec — COMMENTED** [review]
Overall I think it looks really good! It looks like William has given some awesome feedback, and a few changes are necessary

File: src/views/Home/components/About/AboutBox/AboutBox.scss (hunk 1 above)
Comment (krishnakpandian) [reply src/views/Home/components/About/AboutBox/AboutBox.scss]: Just to clarify, are you referring to just min-height or all these lines?

File: src/views/Home/components/About/AboutBox/AboutBox.scss (hunk 4 above)
Comment (krishnakpandian) [reply src/views/Home/components/About/AboutBox/AboutBox.scss]: Oh yeah this seems a lot more natural than the default figma value for a responsiveness.

File: src/views/Home/components/About/AboutBox/AboutBox.scss (hunk 1 above)
Comment (w-gao) [reply src/views/Home/components/About/AboutBox/AboutBox.scss]: Just min-height!

**Review by w-gao — APPROVED** [review]
LGTM. Thanks for making these changes!

**Pull request merged**